packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.ts

```diff
@@ -171,6 +171,10 @@ class CardPayDepositWorkflowTransactionAmountComponent extends Component<CardPay
           'relayTokensTxnReceipt',
           transactionReceipt
         );
+        this.args.workflowSession.update(
+          'depositedAmount',
+          this.amountAsBigNumber.toString() // BN is mutable
```

Comment:
I don't understand the comment. What is the significance of BN being mutable?

Reply:
Since BN is mutable, I was thinking it would be better to add the amount as a string to the shared state of the workflow. We don't mutate it right now, but this is a 'just in case'. Not sure if necesssary.